shacl12-core/index.html

```diff
@@ -4427,7 +6359,7 @@ <h3>sh:defaultValue</h3>
 	sh:property [
 		sh:path ex:postalCode ;
 		sh:name "postal code" ;
-		sh:name "zip code"@en-US ;
+		sh:name "zip code"@en-us ;
```

Comment:
`en-US` is BCP-47 preferred form.

Reply:
The JSON-LD parser makes them lowercase, and rdf-canonize doesn't normalize them. As the JSON-LD from the serializer would have them lowercase, and both styles are allowed, I normalized it. It would be possible to preprocess the language tags to check if both serializations are in sync.